edx/frontend-app-discussions
# [BD-03] [TNL-7414] [BB-2855] Discussion APIs redux integration etc.

This PR adds all the data APIs and redux boilerplate for discussions.

**JIRA tickets**: TNL-7414

**Dependencies**: https://github.com/edx/frontend-app-discussions/pull/1

**Sandbox URL**: NA

**Testing instructions**:

For the purpose of testing the internals of redux have been exposed to allow working via the browser console since this only adds internal APIs. All API methods have been added to window.api, all thunks to window.thunks and the store has been added as well.

For instance you can test the comment adding feature by running the following in your browser console:
``store.dispatch(thunks.addComment("test comment", <thread_id>))``

If you have the comments page open at this time you can see the comment appear in real-time.

**Reviewers**
- [ ] @pcockwell
